microsoft-edge/progressive-web-apps-chromium/how-to/best-practices.md

```diff
@@ -100,7 +100,7 @@ See [Handle files in Progressive Web Apps](./handle-files.md).
 
 Register your app is a link or protocol handler to automatically launch your app when certain links are used in other applications.
 
-See [Handle links to your Progressive Web Apps](./handle-urls.md) and [Handle protocols in Progressive Web Apps](./handle-protocols.md).
+See [Handle links to your Progressive Web App](./handle-urls.md) and [Handle protocols in Progressive Web Apps](./handle-protocols.md).
```

Comment:
```suggestion
See [Handle links to your Progressive Web App](./handle-urls.md) and [Handle protocols in Progressive Web App](./handle-protocols.md).
```

Reply:
this Suggestion seems an error and would be awkward grammar.  actual dest title matches existing link text, "Apps":
```
title: Handle protocols in Progressive Web Apps
description: How to register your PWA as a protocol handler to more deeply integrate it in the operating system with other applications.
# Handle protocols in Progressive Web Apps
```
@captainbrosset

Reply:
i'll check other Comments then close this